sdks/python/test-suites/direct/common.gradle

```diff
@@ -213,8 +213,30 @@ task torchTests {
     }
 }
 
+task sklearnInferenceTest {
+  dependsOn 'installGcpTest'
+  dependsOn ':sdks:python:sdist'
+  doLast {
+      def testOpts = basicTestOpts
+      def argMap = [
+          "test_opts": testOpts,
+          "suite": "postCommitIT-direct-py${pythonVersionSuffix}",
+          "collect": "it_run_inference",
```

Comment:
Do we want to add `it_postcommit`?

Will `it_run_inference` also be needed to be added to the pytorch examples? unless it's specifically for non-pytorch inference tests? Are there concerns with us creating too many of these tags to keep track which ones to use?

Reply:
Yes, it needs to be added to the torch tests. As of now, its not needed because you already have a marker `use_pytorch` which we use in tox environment. 

We are going to create tags for frameworks that needs extra dependencies to run unit tests in the tox environment. 

`it_run_inference ` would be sufficient for all the RunInference IT tests and I think `it_postcommit` is not required anymore. 

Once this change goes in, I will make a change for other torch IT tests but its not a priority as of now

Reply:
I also got confused by this. 

IMO it would be easier to keep consistency with current structure and then replace the with new structure at once if you plan to join pytest and sklearn into one suite

Reply:
made a change to use `uses_sklearn and it_postcommit`